src/pages/_robots.query.graphql

```diff
@@ -0,0 +1,8 @@
+query RobotsTxt {
+  site: _site {
+    noIndex
+  }
+  # app {
+  #   allowAiBots
```

Comment:
I need to create a new DatoCMS environment to add this field. I'll wait for the #230 to be merged as that one introduces the `app` model which this field can be added to.